Sources/SKCore/TaskScheduler.swift

```diff
@@ -20,6 +20,8 @@ public enum TaskDependencyAction<TaskDescription: TaskDescriptionProtocol> {
   case cancelAndRescheduleDependency(TaskDescription)
 }
 
+private let taskSchedulerSubsystem = "org.swift.sourcekit-lsp.task-scheduler"
```

Comment:
Couldn't we just have a separate logger?

Reply:
I just preferred to stick to the `withLoggingSubsystem`, which we’ll also use for indexing (indexing will be in a different subsystem) instead of having two different notions of how to create loggers for subsystems.

And logging needs to store the subsystem in a task local variable because otherwise all log messages from `SwiftPMBuildSystem` that were called from the indexer would end up in the `org.swift.sourcekit-lsp` subsystem instead of `org.swift.sourcekit-lsp.indexing`.